src/core/lib/security/credentials/xds/xds_channel_default_creds.cc

```diff
@@ -1,5 +1,6 @@
 //
-// Copyright 2019 gRPC authors.
+//
+// Copyright 2022 gRPC authors.
```

Comment:
This should keep its existing 2019 copyright date.

Reply:
Craig asked the same question actually. This is a new file that I added last month where I accidentally put 2019, and I'm fixing here. Is that fine?

Reply:
Ah, okay.  Yes, that's fine.